src/content/docs/learn/dusk-evm.md

```diff
@@ -0,0 +1,76 @@
+---
+title: DuskEVM
+description: DuskEVM is a fully EVM-compatible Layer 2 rollup built on the Dusk Network. Powered by the OP Stack and EIP-4844 (Proto-Danksharding), it enables developers to leverage Ethereum tooling while settling on Dusk L1’s compliant, privacy-preserving infrastructure.
```

Comment:
I'm not a big fan of calling DuskEVM a L2 due to the way we position it in our architecture. DuskEVM sits somewhere between a L2 and being an extra execution environment within Rusk. I think we should refer it as one of potentially many execution environments. And the specific layer being the execution layer.

The 7-day challenge period is a quirk we have to live with in the very short term, until we implement our approach to get to same block finality.

Here's an architecture picture to illustrate what I mean: 
![modular-dusk drawio](https://github.com/user-attachments/assets/a6b91680-872a-41cd-b448-f9161446e440)